docs/grpc/javascript.md

```diff
@@ -0,0 +1,62 @@
+# How to write a simple lnd client in Javascript using nodejs?
+
+First, you need to initialize a simple nodejs project:
+
+```
+npm init (or npm init -f if you want to use the default values without prompt)
+```
+
+Then you need to install the Javascript grpc library dependency:
+
+```
+npm install grpc --save
+```
+
+You also need to copy the `lnd` `rpc.proto` file in your project directory (or at least somewhere reachable by your Javascript code).
+
+The `rpc.proto` file is located in the `lnrpc` directory of the `lnd` sources.
+
+I had to comment the following line in the `rpc.proto` file otherwise `grpc` would crash:
+
+```
+//import "google/api/annotations.proto";
```

Comment:
Hmm, this looks related to: https://github.com/dcodeIO/protobuf.js/issues/433

We can either investigate a direct fix, or just re-word the statement above to be declarative:
>  In order to use the protos from `js`, you'll need to comment out the following line.